IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -5711,7 +5782,35 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		1EE8FF6124F4BEDB00CA1445 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n# target directory is Project root directory\ntarget_dir=${SRCROOT}/include/\n# Ensure the target include path exists\nmkdir -p ${target_dir}\n# Copy any file that looks like a Swift generated header to the include path\ncp ${DERIVED_SOURCES_DIR}/*-Swift.h ${target_dir}\n";
```

Comment:
Ok, so this filters out all Swift files and copies them to include folder? Any context on why this is needed?

Reply:
It is how xcode creates a Objective header for the swift file and adds it to the derived sources folder under the IdentityCoreSwift target. We need a script to move that file to the SOURCE_DIR so the IdentityCoreTests can reference it in the header search paths and use it accordingly.

Reply:
Please refer to this .https://medium.com/@mail2ashislaha/swift-objective-c-interoperability-static-libraries-modulemap-etc-39caa77ce1fc